common/djangoapps/util/enterprise_helpers.py

```diff
@@ -29,6 +33,23 @@ def enterprise_enabled():
     return 'enterprise' in settings.INSTALLED_APPS
 
 
+def consent_needed_for_course(user, course_id):
+    if not enterprise_enabled():
+        return False
+    return consent_necessary_for_course(user, course_id)
+
+
+def redirect_to_course_specific_consent(request, course_id):
```

Comment:
I'd expect a function named like this to actually perform a redirect, not merely to return a URL.  How about `get_course_specific_consent_url()` or somehting similar?